loadtest/control/actions.go

```diff
@@ -46,6 +47,8 @@ func SignUp(u user.User) UserActionResponse {
 	return UserActionResponse{Info: fmt.Sprintf("signed up as %s", username)}
 }
 
+// Login authenticates with the server and fetches teams, users and channels that
```

Comment:
I am wondering whether we need to mention the store population logic at all. It is there in nearly every action, and then we would need to do it everywhere. I would suggest to just go with "Login logs in a user."

Reply:
I think this kind if comments can be useful for developers who are using this code as a library. Also `Login(user)` already tells that user us logging in. We can think it like an additional metadata.